### DISCUSSION TITLE
Error reported after "make"

### DISCUSSION THREAD

**Discussion by KangChenRui** [idaholab/moose#22510] [Q&A General] [answered]
Hello everyone!
Error reported after "make",But it worked fine yesterday. I don't konw what happened.
Please help me!
![image](https://user-images.githubusercontent.com/91399933/198174195-58eea914-a484-49be-a4b0-82566131ea2d.png)
![image](https://user-images.githubusercontent.com/91399933/198174250-0a829e28-e0f7-419d-ab19-f449ed7da4bf.png)
![image](https://user-images.githubusercontent.com/91399933/198174421-ee073480-d7d9-4f56-9623-c332801a9129.png)
![image](https://user-images.githubusercontent.com/91399933/198174452-0ec4819c-3e9b-4d1f-9ab0-cafd01498ea9.png)
![image](https://user-images.githubusercontent.com/91399933/198174507-766439be-3a65-4e90-a950-f39455fda05e.png)
![image](https://user-images.githubusercontent.com/91399933/198174568-8381630d-65b2-4d6c-b063-d5c9bdc3525e.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

Seems odd. Did you just udpate MOOSE?
Can you make clobberall before building?

Guillaume

**Reply by KangChenRui** [reply]
I did not update MOOSE.

**Reply by GiudGiud** [reply]
please try cleaning the repository then building again
